tests/test_check.py

```diff
@@ -0,0 +1,24 @@
+"""Run always from the `pyta` root directory, so the local `python_ta` is
+used rather than the installed `python_ta` package.
+"""
+import unittest
+import os
+import sys
+import python_ta
+
+_NODES_PATH_TOP_LEVEL = 'nodes'
+
+
+class TestCheck(unittest.TestCase):
```

Comment:
Whoops, didn't notice this one earlier. We aren't using `unittest`; you can write a top-level `test` function for running with `nosetests`.